launch: Region Pages w/ Needs Assessment Viz

jtfairbank opened the pull request:
Makes progress on:

  - #812

jtfairbank committed:
feat: Add new fields from forestry to Region and Subregion

ghost commented:
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=912&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=912&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org)
</details>

[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/distributeaid/distributeaid.org/912/37d5dcba/a27050308b25a2e9bda3018556e43dbfdd452eae.svg)](https://app.codesee.io/r/reviews?pr=912&src=https%3A%2F%2Fgithub.com%2Fdistributeaid%2Fdistributeaid.org)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

jtfairbank committed:
Update from Forestry.io - Updated Forestry configuration

Commit:
Update from Forestry.io - Updated Forestry configuration

jtfairbank committed:
chore: touchup needs assessment methodology content location

jtfairbank committed:
feat: Load the 2023 Q1 needs assessment data.

jtfairbank committed:
Update from Forestry.io
Taylor Fairbank updated content/pages/regions/lebanon/bekka-valley.md

jtfairbank committed:
Update from Forestry.io
Taylor Fairbank updated content/pages/regions/lebanon/beirut.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/serbia.md

jtfairbank committed:
Update from Forestry.io - Updated Forestry configuration

jtfairbank committed:
Update from Forestry.io - Updated Forestry configuration

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/croatia.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/bosnia.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/lebanon/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/lebanon/bekka-valley.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/northern-greece.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/southern-greece.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/aegean-islands.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/france/northern-france.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/ukraine-subregion-map.png

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/slovakia.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/ukraine-subregion-map.png
Distribute Aid Admin updated content/pages/regions/eastern-europe/ukraine.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/romania.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/poland.md
Distribute Aid Admin created static/uploads/slovakia-subregion-map.png

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/slovakia.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/romania-subregion-map.png
Distribute Aid Admin updated content/pages/regions/eastern-europe/romania.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/moldova.md
Distribute Aid Admin created static/uploads/nothern-greece-subregion-map.png
Distribute Aid Admin created static/uploads/poland-subregion-map.png

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/poland.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/moldova-subregion-map.png
Distribute Aid Admin updated content/pages/regions/eastern-europe/moldova.md
Distribute Aid Admin updated content/pages/regions/eastern-europe/hungary.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/hungary-subregion-map.png
Distribute Aid Admin updated content/pages/regions/eastern-europe/hungary.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/northern-france-subregion-map.png
Distribute Aid Admin updated content/pages/regions/france/northern-france.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/southern-greece-subregion-map.png
Distribute Aid Admin updated content/pages/regions/greece/southern-greece.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/northern-greece.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/aegean-islands-subregion-map.png
Distribute Aid Admin updated content/pages/regions/greece/aegean-islands.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/bekka-subregion-map.png

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/lebanon/bekka-valley.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/beirut-subregion-map.png
Distribute Aid Admin updated content/pages/regions/lebanon/beirut.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/serbia-subregion-map.png
Distribute Aid Admin updated content/pages/regions/the-balkans/serbia.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/croatia-subregion-map.png
Distribute Aid Admin updated content/pages/regions/the-balkans/croatia.md

Commit:
Update from Forestry.io
Distribute Aid Admin created static/uploads/bosnia-subregion-map.png

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/croatia.md

jtfairbank committed:
fix: remove accidental punctuation from beirut's population.count field

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/bosnia.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/bosnia.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/lebanon/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/lebanon/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/serbia.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/croatia.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/bosnia.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/croatia.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/serbia.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/index.md

jtfairbank committed:
feat: add regions to the main menu, improve region & subregion page titles

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/lebanon/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/lebanon/bekka-valley.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/lebanon/beirut.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/croatia.md

jtfairbank committed:
feat: add needs to the main menu, setup needs explorer page.

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/northern-greece.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/southern-greece.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/aegean-islands.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/france/northern-france.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/france/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/france/index.md

jtfairbank committed:
feat: style the regions page

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/ukraine.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/slovakia.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/romania.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/poland.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/moldova.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/hungary.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/needs-assessments/methodology.md

Commit:
Update from Forestry.io
Distribute Aid Admin created content/pages/needs-assessment-overview.md

jtfairbank committed:
Update from Forestry.io
Taylor Fairbank created content/pages/needs-assessments/test.md

jtfairbank committed:
fix: clean up needs assessment methodology content

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/needs-assessments/methodology.md

Commit:
Update from Forestry.io - Updated Forestry configuration

Commit:
Update from Forestry.io - Updated Forestry configuration

jtfairbank committed:
fix: touch up nav links on regions / subregions pages.

jtfairbank committed:
Update from Forestry.io
Taylor Fairbank updated content/pages/needs-assessments/methodology.md

jtfairbank committed:
Update from Forestry.io
Taylor Fairbank updated content/pages/needs-assessments/methodology.md

jtfairbank committed:
Update from Forestry.io
Taylor Fairbank updated content/pages/needs-assessments/methodology.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/needs-assessments/methodology.md

Commit:
Update from Forestry.io
Distribute Aid Admin created content/pages/needs-assessments/needs-assessment-overview.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/needs-assessments/needs-assessment-overview.md

jtfairbank committed:
hack: use generic section / block test factories to hack in generic content support for custom pages

jtfairbank committed:
Update from Forestry.io - Updated Forestry configuration

jtfairbank committed:
Update from Forestry.io - Updated Forestry configuration

Commit:
Update from Forestry.io - Updated Forestry configuration

Commit:
Update from Forestry.io - Updated Forestry configuration

jtfairbank committed:
fix: rename the needs-assessment/overview page

jtfairbank committed:
feat: add link to needs-assessment/methodology in the sub-menu

jtfairbank committed:
feat: enable frontent Blocks for LinksList and UpdatesList, apply to regions and subregions pages.

jtfairbank committed:
feat: clean up region / subregoin page styles

jtfairbank committed:
fix: some locale-dependent test data for updates list dates

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/needs-assessments/methodology.md

hola-soy-milk committed:
fix: Tweak region and subregion layout

Co-authored-by: jennjunod
Co-authored-by: Jarvis1010

hola-soy-milk committed:
docs: Add Jenn and Travis

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/needs-assessments/overview.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/needs-assessments/overview.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/romania.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/moldova.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/eastern-europe/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/france/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/france/northern-france.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/aegean-islands.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/northern-greece.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/greece/southern-greece.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/lebanon/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/lebanon/bekka-valley.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/lebanon/beirut.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/index.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/bosnia.md

Commit:
Update from Forestry.io
Distribute Aid Admin updated content/pages/regions/the-balkans/serbia.md

jtfairbank committed:
feat: regions / subregions will use needs link from Forestry if it's present.

jtfairbank committed:
feat: adjust needs explorer layout, fix needs overview data issues, update region / subregion needs links

jtfairbank committed:
fix: failing tests due to re-introduced needs bar chart sort order bug

jtfairbank committed:
feat: update main menu

jtfairbank committed:
feat: add current need to regions page

jtfairbank committed:
feat: mobile styles for regions and needs pages.
